src/index.js

```diff
@@ -38,4 +39,8 @@ angular.module('vlui', [
       T: 'time',
       G: 'geo'
     }
+  })
+  .config(function (AnalyticsProvider) {
+    AnalyticsProvider
+      .setAccount({ tracker: 'UA-44428446-4', name: 'voyager', trackEvent: true });
```

Comment:
yeah. i think this and the myria customization would lend themselves to a
config file approach - which could be customized per installation.

On Wed, Sep 30, 2015 at 8:18 AM, K.Adam White <notifications@github.com>
wrote:

> In src/index.js
> <https://github.com/vega/vega-lite-ui/pull/83#discussion_r40807037>:
>
> > @@ -38,4 +39,8 @@ angular.module('vlui', [
> >        T: 'time',
> >        G: 'geo'
> >      }
> > +  })
> > +  .config(function (AnalyticsProvider) {
> > +    AnalyticsProvider
> > +      .setAccount({ tracker: 'UA-44428446-4', name: 'voyager', trackEvent: true });
>
> My only concern here is that this will make all consumers of vega-lite-ui
> use this tracking ID; it's not hugely likely for now that third parties
> would be using these tools, but since there's no formal reason why they
> would not be able to, this feels like it has the potential to mess up our
> stats.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-lite-ui/pull/83/files#r40807037>.
>